src/app/zap-templates/zcl/data-model/chip/mode-base-cluster.xml

```diff
@@ -86,8 +86,7 @@ This is because zap does not currently support generating code for clusters that
 <!--    <cluster code="0x..."/>-->
     <cluster code="0x0051"/>
     <cluster code="0x0052"/>
-    <cluster code="0x0054"/>
-    <cluster code="0x0055"/>
```

Comment:
This is a breaking change for Darwin.  This is removing not just the feature bit (which was communicated as likely to be removed and hence never shipped) but also the Feature bitmap definition itself from these clusters.  

That is, from Darwin's pov the bare minimum here is that a Feature definition needs to exist for these two clusters, with no bits defined.